index.html

```diff
@@ -1,30 +1,62 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Awesome Books</title>
-    <link rel="stylesheet" href="style.css" />
-  </head>
-  <body>
-    <header>
-      <nav>
-        <p class="logo">Awesome books</p>
-        <ul>
-          <li><a href="index.html">List</a></li>
-          <li><a href="AddBook.html">Add New</a></li>
-          <li><a href="contact.html">Contact</a></li>
-        </ul>
-      </nav>
-    </header>
-    <h1>Awesome books</h1>
-    <section class="record"></section>
 
-    <footer>
-      <div class="logo">Awesome Books</div>
-      <p> &copy; All rights are reserved</p>
-    </footer>
-    <script src="main.js"></script>
-  </body>
-</html>
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Awesome Books</title>
+  <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+  <header>
+    <nav>
+      <p class="logo">Awesome books</p>
+      <ul>
+        <li><a href="#" id="booksList">List</a></li>
+        <li><a href="#" id="newBook">Add New</a></li>
+        <li><a href="#" id="contactInfo">Contact</a></li>
+      </ul>
+    </nav>
+  </header>
+  <div class="date-and-time">
+    <p id="current-date">position for date & time!</p>
+  </div>
```

Comment:
- [ ] The current date Should be displayed below the navigation bar. Currently, nothing is displayed. Let's add it
<img width="1440" alt="Screenshot 2022-12-10 at 12 56 20 AM" src="https://user-images.githubusercontent.com/40503704/206813766-ccb5676e-bed6-4828-b770-208e70ab7c30.png">